packages/p2p/src/pages/orders/chat/chat-messages.jsx

```diff
@@ -73,63 +84,63 @@ const ChatMessages = observer(() => {
                 refSetter={scroll_ref}
                 onScroll={event => sendbird_store.onMessagesScroll(event)}
             >
-                {sendbird_store.chat_messages.map(chat_message => {
-                    const is_admin_message = chat_message.custom_type === ChatMessage.TYPE_ADMIN;
-                    const is_my_message =
-                        chat_message.sender_user_id === sendbird_store.chat_info.user_id && !is_admin_message;
-                    const message_date = formatMilliseconds(chat_message.created_at, 'MMMM D, YYYY');
-                    const message_color = is_my_message ? 'colored-background' : 'general';
-                    const should_render_date =
-                        current_date !== message_date && Boolean(!is_admin_message && (current_date = message_date));
+                <React.Fragment>
```

Comment:
I wonder if the [new eslint rule](https://github.com/binary-com/deriv-app/pull/10327) will break this?
```suggestion
                <>
```

Can we use the above syntax?

Reply:
@nada-deriv did it break? I believe that rule is only for wallets package.

Reply:
no.. it didn't break actually.. but i did make the change..